.gitattributes

```diff
@@ -1,3 +1,3 @@
-* text=auto
+* text=auto eol=lf
```

Comment:
For reference: https://github.com/prettier/prettier/issues/7853#issuecomment-603843422